### DISCUSSION TITLE
Clean-up process

### DISCUSSION THREAD

**Discussion by Adu2120** [vercel/turborepo#8117] [Help]
### Summary

I'm using turbo for my TypeScript project. 
I want to do fresh build. So, I'm cleaning up my node_modules and build folders. but as turbo runs daemon thread in background, if I kill that daemon thread and clean the build folders, then will that affect anywhere in my project?

**Comment by anthonyshew** [comment]
From what I understand, I don't believe the daemon will have any effect on the workflow you're describing.

Did you give it a try? I'd be interested to learn if you found any unwanted behaviors.

**Comment by gylove1994** [comment]
I am using PM2 for local development of a moti instance, so I need to use 3 commands:

```text
    "pm2:dev": "dotenvx run -- pm2 start ecosystem.config.js",
    "pm2:logs": "pm2 logs",
    "pm2:delete": "pm2 delete ecosystem.config.js",
```

Start with `pm2:dev & pm2:logs`

End with `pm2:delete`

So that's why I think having a clean-up process would be great.
